test/src/main/java/org/apache/accumulo/test/ShellServerIT.java

```diff
@@ -1939,6 +1939,16 @@ public void scans() throws Exception {
     ts.exec("deletetable -f t");
   }
 
+  @Test
+  public void scansWithColon() throws Exception {
+    ts.exec("createtable twithcolontest");
+    make10WithColon();
```

Comment:
```suggestion
    ts.exec("insert row c:f cq value")
```
This test really only needs one row in the table.